src/types/resolved.ts

```diff
@@ -6,5 +6,6 @@ export type Resolved<T extends Route> = {
   name: string,
   path: string,
   regex: RegExp,
+  parentNames: T['name'][],
```

Comment:
this feels a bit awkward to me. But I see why you went this direction. But I don't think this is something we'll want as part of the libraries public types and we've talked about `Resolved<Route>` being the type that is used for middleware and such. This seems like an implementation detail and not something we'd actually give to the user. Any thoughts on that?

Reply:
agreed. Let's include this in our conversation about types this afternoon.